sql/common/src/test/scala/org/apache/sedona/sql/functionTestScala.scala

```diff
@@ -137,6 +137,20 @@ class functionTestScala extends TestBaseScala with Matchers with GeometrySample
       assert(functionDf.count() > 0);
     }
 
```

Comment:
You can follow the map way of specifying multiple test cases and their expected output here, it is more readable